src/tests/multihost/sssd/testlib/common/utils.py

```diff
@@ -732,6 +734,25 @@ def dump_ldb(self, entity, domain):
                         ldb_info[item_name] += lines[idx + 1].strip()
         return ldb_info
 
+    @staticmethod
```

Comment:
If You use the object (not a staticmethod), it has a multihost instance so there should be the hostname somewhere so it would not be needed as a parameter.